.github/workflows/rust.yml

```diff
@@ -58,16 +58,19 @@ jobs:
     - uses: actions/checkout@v3
     - uses: stellar/actions/rust-check-git-rev-deps@main
 
-  semver-checks:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v3
-    - run: rustup update
-    - uses: stellar/binaries@v29
-      with:
-        name: cargo-semver-checks
-        version: 0.35.0
-    - run: cargo semver-checks --exclude soroban-simulation
+  # This check is currently broken due to incompatible `arbitrary` crate 
```

Comment:
Rather than remove the job I think we should remove it from the needs list on the `complete` job. That way we know when it starts passing again but it won't block builds.

Reply:
Makes sense, done.